packages/react-google-maps-api/src/components/streetview/StreetViewPanorama.js

```diff
@@ -73,13 +73,12 @@ export class StreetViewPanorama extends PureComponent {
 
   componentDidMount = () => {
     const streetViewPanorama = this.context.getStreetView()
-
     this.setState(
       () => ({
         streetViewPanorama
       }),
       () => {
-        this.state.streetViewPanorama.setVisible(true)
```

Comment:
Is this change intentional?

Reply:
Not really. StreetViewPanorama is not finished yet and is not available for use. I made some changes to it before lerna and they went to PR accidentally